web/beacon-app/src/components/ui/SetupTenant/CustomizeTenant.tsx

```diff
@@ -0,0 +1,23 @@
+import { Button } from '@rotational/beacon-core';
+
+import world from '/src/assets/images/world-icon.png';
+
+export default function CustomizeTenant() {
+  return (
+    <div className="mx-auto max-w-2xl rounded-lg border border-solid border-primary-800">
+      <div className="max-w-sm p-10 text-center">
+        <h3 className="text-xl font-bold">Customize Tenant</h3>
+        <img src={world} alt="" className="mx-auto mt-5" />
+        <p className="mt-6">
+          You can customize your tenant settings, including regions (single and multi-region) and
+          cloud providers.
+        </p>
+        <p className="mt-6">
+          Customizing your tenant requires a paid plan. Upgrade now to set up a tenant specific to
+          your development and modeling needs.
+        </p>
+        <Button className="mt-16 h-14 w-48 rounded bg-danger-500 text-center text-xl font-bold text-white">Coming Soon</Button>
```

Comment:
I created a class for the repeated button styles in the CSS file, but the styling didn't appear in Storybook so I added it back to the className here.